### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by ouji-san** [finndev/PokeBuddy#1048]
Well I have a problem. The problem is when I login my PTC they always live a message of  "[!] Access token expired. Reauthenticating..." and they doesn't give me XP in the pokestops and when walking around whit the exe. And I need to have pokeballs, incense  lucky eggs... please somebody help me.

**Comment by Kokainium** [comment]
Maybe softban ?

**Comment by AlexCtasy** [comment]
Yes it's softban, so u need to wait 1-12hs if u still have access to ur account.

**Comment by ouji-san** [comment]
I don't understand

**Comment by ouji-san** [comment]
they take to long to get XP and caught pokemons

**Comment by ZzYaPoO** [comment]
Pokemon go at this few day create alot policy for us, we need to try 1 by 1and slowly how about their policy.
my main account already banned, i try more way to use bot and get more pokemon, i need start from 0 again, that really FXXX...
:(

**Comment by ouji-san** [comment]
Shit ok I have to stop use this exe. i don't whant be ban

**Comment by AlexCtasy** [comment]
Stop with boting for now if u want to keep ur account. Niantic is just installed maybe on servers something that are detecting all bots app, i 100 sure in that sh1t is included the developers of bots app helping then.
